src/convert/from_h5ad_to_h5mu/test.py

```diff
@@ -34,8 +35,10 @@ def test_run(run_component, tmp_path):
 
     mdata2 = mu.read_h5mu(tmp_output)
 
-    assert "rna" in mdata2.mod, "Resulting mudata should contain rna modality"
-    assert "prot" in mdata2.mod, "Resulting mudata should contain rna modality"
+    assert list(mdata2.mod.keys()) == ["rna", "prot"]
+    
+    assert_annotation_objects_equal(mdata2.mod["rna"], str(tmp_rna))
```

Comment:
Does the `assert_annotation_objects_equal` method not work without using `str`? Otherwise I would suggest to update it so that it does.